integration/spark/shared/build.gradle

```diff
@@ -44,7 +43,6 @@ dependencies {
     api("io.openlineage:openlineage-java:${project.version}")
     api("io.openlineage:openlineage-sql-java:${project.version}")
     api("io.openlineage:spark-interfaces-scala_${scalaBinaryVersion}:${project.version}")
-    api("io.micrometer:micrometer-core:${micrometerVersion}")
```

Comment:
no need to add micrometer in each subproject as it's already in openlineage-java.